ops/pebble.py

```diff
@@ -1071,11 +1078,13 @@ def from_dict(cls, d: _ServiceInfoDict) -> ServiceInfo:
         try:
             startup = ServiceStartup(d['startup'])
         except ValueError:
-            startup = d['startup']
+            warnings.warn(f'Unknown ServiceStartup value {d["startup"]!r}')
```

Comment:
It would be nice if these warnings helped guide people towards solving the problem somehow. I assume our expectation here is that the charm is using a version of Pebble that is more recent than the version of ops knows of, and some new value was added to Pebble (rather than someone is passing in data that didn't come from Pebble, for example).

So perhaps the solution should generally be "update ops"? In which case we could perhaps say something like "try updating to the latest version of ops"?

Reply:
@benhoyt what do you think about changing these warning messages to provide problem solving help, and if so what would be the right message? "Unknown value blah, the ops version may be lower than the pebble version" or something?

Reply:
Yeah, I think something (succinct) like that would be good. Perhaps don't compare ops version with pebble version, as those aren't related. Maybe what Tony said, or "; do you need to update ops?"